Wikipedia/Code/MWKArticle.m

```diff
@@ -357,7 +357,8 @@ - (MWKProtectionStatus *)requiredProtectionStatus:(NSString *)key dict:(NSDictio
             }
         }
     }
-    return imageURLs;
+
+    return [[NSOrderedSet orderedSetWithArray:imageURLs] array];
```

Comment:
I think the de-dupe could (should?) probably happen inside the 2 methods in `WMFImageTagList+ImageURLs.m` instead...